### DISCUSSION TITLE
Radiation detector

### DISCUSSION THREAD

**Discussion by FarrunDragon** [FakeFishGames/Barotrauma#8811] [Ideas and suggestions]
I think a radiation detector would be nice; just a simple thing like a motion or water detector. Outputs of state out, number of rads/second, that kind of thing.

**Comment by Hazmat616** [comment]
Geiger counter

**Comment by pavel4win** [comment]
I agreed with topic starter.
There is no way to determine radiation except for diagnosing radiation sickness. 
Introduce a radiation detector and a hand-held radiation measuring device, please.
